apps/files_external/lib/Controller/StoragesController.php

```diff
@@ -317,37 +317,16 @@ protected function updateStorageStatus(StorageConfig &$storage, $testOnly = true
 	 * @return DataResponse
 	 */
 	public function index() {
-		$storages = $this->formatStoragesForUI($this->service->getStorages());
+		$storages = array_map(function ($storage) {
+			return $storage->jsonSerialize(true);
+		}, $this->service->getStorages());
```

Comment:
```suggestion
		$storages = array_map(fn ($storage) => $storage->jsonSerialize(true), $this->service->getStorages());
```
Short call syntax is made for this kind of stuff